app/assets/stylesheets/components/_onboarding_message.scss

```diff
@@ -0,0 +1,5 @@
+.onboarding-message {
+  text-align: center;
+  margin: 0;
+  padding: $baseline-unit*6 0;
+}
```

Comment:
Probably a stupid question but does this cater for the styling with the light green box per the screenshot?

Also is this completely separate to the flash messages?